### PR TITLE
Invoice list UI

### PR DESCRIPTION
**Pull request by NIhalJ216**

### CODE REVIEW

File: web/src/components/authentication/login/LoginForm.js [hunk 1]
```diff
@@ -126,7 +128,9 @@ export default function LoginForm() {
             control={<Checkbox {...getFieldProps('remember')} checked={values.remember} />}
             label="Remember me"
           />
-          <Link variant="subtitle2">Forgot password?</Link>
+          <Link component={RouterLink} variant="subtitle2" to={PATH_AUTH.resetPassword}>
```
Comment (sonawane-sanket) [inline web/src/components/authentication/login/LoginForm.js]: Removed Login page related changes

Comment (sonawane-sanket) [reply web/src/components/authentication/login/LoginForm.js]: Also removed forgotPassfordForm.js

File: web/src/components/table/simpleTable.js [hunk 2]
```diff
@@ -356,7 +271,7 @@ export default function SimpleTable({
       )}
       <Grid item xs={12} lg={12}>
         <DataTable
-          header={header}
+          header={isglobalfilter ? header : null}
```
Comment (sonawane-sanket) [inline web/src/components/table/simpleTable.js]: isglobalfilter should be camelcase =>  isGlobalFilter

File: web/src/pages/Administration/CountryList/countrylist.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.Country_List_main_cls{
```
Comment (sonawane-sanket) [inline web/src/pages/Administration/CountryList/countrylist.scss]: Class name should be in camelcase or hyphen seperated
E.X. = redBox or red-box

File: web/src/pages/InvoiceList/TablePinTrial.js [hunk 4]
```diff
@@ -122,7 +122,7 @@ function TablePinTrial() {
         </Grid>
         <Grid item xs={12}>
           <SimpleTable
-            rowData={InvoiceListingData}
+            rowData={InvoiceData}
```
Comment (sonawane-sanket) [inline web/src/pages/InvoiceList/TablePinTrial.js]: should be camescase => invoiceData

File: web/src/pages/auth/ForgotPassword.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```
Comment (sonawane-sanket) [inline web/src/pages/auth/ForgotPassword.js]: Removed this page

File: web/src/components/DataTable.js [hunk 6]
```diff
@@ -89,101 +99,240 @@ function ContractList({
...
+            />
+          </div>
+        );
+      default:
+        return undefined;
+    }
+  };
+  // Handle ExpandedDatechange
+  const handleExpandedChangeDate = (key, value, options) => {
+    // console.log('Logging...', newValue, data, key);
```
Comment (sonawane-sanket) [inline web/src/components/DataTable.js]: removed unnecessary commented code

File: web/src/pages/InvoiceList/InvoiceList.js [hunk 7]
```diff
@@ -1,31 +1,81 @@
...
+import SaveIcon from '@mui/icons-material/Save';
+import EditIcon from '@mui/icons-material/Edit';
+import PrintIcon from '@mui/icons-material/Print';
+import Filters from '../../components/Filter/filter';
+import { COMPONENTS } from '../../utils/constants';
+import { SEVICE_DASHBOARD_FILTER_MASTER_DATA } from '../../components/ServiceBoard/data';
+import { POST_OFFICE } from '../../redux/constants';
 import SimpleTable from '../../components/table/simpleTable';
+import { InvoiceData } from './Data';
+import './InvoiceList.scss';
```
Comment (sonawane-sanket) [inline web/src/pages/InvoiceList/InvoiceList.js]: filename should be in camelcase

File: web/src/pages/InvoiceList/InvoiceList.js [hunk 8]
```diff
@@ -154,77 +237,35 @@ function InvoiceList() {
...
+          />
         </Grid>
-        {/* Filter section xs={12} end */}
       </Grid>
       {/* Grid for simple table */}
       <Grid container spacing={3} style={{ marginTop: '1rem' }}>
         <Grid item xs={12}>
           <SimpleTable
-            rowData={InvoiceListingData}
+            rowData={InvoiceData}
```
Comment (sonawane-sanket) [inline web/src/pages/InvoiceList/InvoiceList.js]: should use camelcase consistency => invoiceData

**Review by sonawane-sanket — CHANGES_REQUESTED** [review]
Comments added

File: web/src/components/authentication/login/LoginForm.js (hunk 1 above)
Comment (NIhalJ216) [reply web/src/components/authentication/login/LoginForm.js]: Updated

File: web/src/components/table/simpleTable.js (hunk 2 above)
Comment (NIhalJ216) [reply web/src/components/table/simpleTable.js]: Updated

File: web/src/pages/Administration/CountryList/countrylist.scss (hunk 3 above)
Comment (NIhalJ216) [reply web/src/pages/Administration/CountryList/countrylist.scss]: Updated

File: web/src/pages/InvoiceList/TablePinTrial.js (hunk 4 above)
Comment (NIhalJ216) [reply web/src/pages/InvoiceList/TablePinTrial.js]: Updated

File: web/src/pages/auth/ForgotPassword.js (hunk 5 above)
Comment (NIhalJ216) [reply web/src/pages/auth/ForgotPassword.js]: Removed Page

File: web/src/components/DataTable.js (hunk 6 above)
Comment (NIhalJ216) [reply web/src/components/DataTable.js]: updated

File: web/src/pages/InvoiceList/InvoiceList.js (hunk 7 above)
Comment (NIhalJ216) [reply web/src/pages/InvoiceList/InvoiceList.js]: Updated

File: web/src/pages/InvoiceList/InvoiceList.js (hunk 8 above)
Comment (NIhalJ216) [reply web/src/pages/InvoiceList/InvoiceList.js]: Updated

File: web/src/pages/auth/ForgotPassword.js (hunk 5 above)
Comment (sonawane-sanket) [reply web/src/pages/auth/ForgotPassword.js]: Not resolved

File: web/src/pages/InvoiceList/TablePinTrial.js (hunk 4 above)
Comment (sonawane-sanket) [reply web/src/pages/InvoiceList/TablePinTrial.js]: not resolved

File: web/src/components/DataTable.js (hunk 6 above)
Comment (sonawane-sanket) [reply web/src/components/DataTable.js]: not resolved

File: web/src/pages/InvoiceList/InvoiceList.js (hunk 7 above)
Comment (sonawane-sanket) [reply web/src/pages/InvoiceList/InvoiceList.js]: Not resolved

File: web/src/pages/InvoiceList/TablePinTrial.js (hunk 4 above)
Comment (NIhalJ216) [reply web/src/pages/InvoiceList/TablePinTrial.js]: I deleted this page, it will not needed

File: web/src/components/DataTable.js (hunk 6 above)
Comment (NIhalJ216) [reply web/src/components/DataTable.js]: that colsole line is removed I'll commit again

File: web/src/pages/InvoiceList/InvoiceList.js (hunk 7 above)
Comment (NIhalJ216) [reply web/src/pages/InvoiceList/InvoiceList.js]: file name changed

File: web/src/pages/auth/ForgotPassword.js (hunk 5 above)
Comment (NIhalJ216) [reply web/src/pages/auth/ForgotPassword.js]: Page is deleted

**Pull request merged**